src/server/launchpad/resources.js

```diff
@@ -75,10 +85,18 @@ export class Collection extends Resource {
 export class Entry extends Resource {
   constructor(client, representation, uri) {
     super();
-    this.lp_client = client;
-    this.uri = uri;
-    this.lp_attributes = {};
-    this.dirty_attributes = [];
+    this.init(client, uri);
```

Comment:
`lp_client` and `uri` will be inherited from `Resource` so will be non-enumerable.